content/millennium/bulk-data.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Bulk Data Access
+layout: api
+---
+
+# Bulk Data Access Overview
+
+The Cerner Ignite APIs for Millennium with Bulk Data Access provides system applications the ability to access large volumes of core clinical data for a group of individuals in alignment with standards cited in 21st Century Cures Act regulations. 
+
+FHIR Bulk data requests are executed against an organization's database asynchronously. When implementing an interface between a Cerner Millennium domain and a bulk data app, it is important that the queries inherent in the bulk data request capture only the data necessary for the application.
+
+There are several factors that affect the amount of time it takes to complete a Bulk API export:
+
+* The size of the population for which data was requested
+* The number of resource types requested
+* The time range of activity being requested
+* Inherent latency due to network issues and effects from concurrent requests to the system
+
+#### Example use cases
+
+* Creating an internal clinical data warehouse for study purposes
+* Consolidation of claims data for comprehensive view
```

Comment:
Although a valid use case, we don't support the common financial data set use case documented [here](https://hl7.org/fhir/uv/bulkdata/index.html#common-financial-data-set) so maybe we should leave this out?

Reply:
I removed it: https://github.com/cerner/fhir.cerner.com/pull/899/commits/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1L22